src/syrupy/location.py

```diff
@@ -28,7 +28,8 @@ def __post_init__(self) -> None:
         self.__attrs_post_init_def__()
 
     def __attrs_post_init_def__(self) -> None:
-        self.filepath = getattr(self._node, "fspath")  # noqa: B009
+        node_path: Path = getattr(self._node, "path")  # noqa: B009
+        self.filepath = str(node_path.absolute())
```

Comment:
Simplest to just convert Path to a string for now, can be cleaned up later to just use Path objects everywhere